classes/class.ilMumieTaskUserListGUI.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/**
+ * MumieTask plugin
+ *
+ * @copyright   2019 integral-learning GmbH (https://www.integral-learning.de/)
+ * @author      Tobias Goltz (tobias.goltz@integral-learning.de)
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+/**
+ * This form is used to add, edit and validate MUMIE Server configurations
+ *
+ */
+class ilMumieTaskUserListGUI extends ilTable2GUI
+{
+    private $members;
+    private $parent_gui;
+    private $postvar;
+
+    public function __construct($parentObj, $form)
```

Comment:
this function is pretty huge for a simple constructor. Constructors are usually used to initialzie important variables whose values are given as parameters.

Could you please put the remaining logic in a new function called init. That's more consistent with the ilias way of coding.

Just have a look at ilMumieTaskServerTableGUIfor comparison

Reply:
done